packages/trpc/server/routers/viewer.tsx

```diff
@@ -1176,6 +1175,35 @@ const loggedInViewerRouter = router({
         });
       }
     }),
+  getUsersDefaultConferencingApp: authedProcedure.query(async ({ ctx }) => {
+    return userMetadata.parse(ctx.user.metadata)?.defaultConferencingApp;
+  }),
+  updateUserDefaultConferencingApp: authedProcedure
+    .input(
+      z.object({
+        appSlug: z.string().optional(),
+        appLink: z.string().optional(),
+      })
+    )
+    .mutation(async ({ ctx, input }) => {
+      const currentMetadata = userMetadata.parse(ctx.user.metadata);
+
+      await ctx.prisma.user.update({
+        where: {
+          id: ctx.user.id,
+        },
+        data: {
+          metadata: {
+            ...currentMetadata,
+            defaultConferencingApp: {
```

Comment:
Could be a NIT for now, but I think we should add the appLink configuration to the InstalledApp, e.g. if I am a Zoom user in my team, I would like to have a per-user app link and this would ideally be configurable through the app configuration.

Reply:
You mean handling the default link of an app on the installed app itsself? 

Thats deffo an option we could do down the road - I think that makes more sense when we come to implementing managed event types. Happy to implement now if you think its needed